src/ResponsiveProvider.js

```diff
@@ -151,16 +151,28 @@ ResponsiveProvider.propTypes = {
         PropTypes.arrayOf(PropTypes.node),
         PropTypes.node,
     ]).isRequired,
-    breakpoints: PropTypes.arrayOf(
-        PropTypes.shape({
-            mediaQuery: PropTypes.string,
-            mediaType: PropTypes.string,
-        })
-    ),
-    // eslint-disable-next-line react/forbid-prop-types
-    mediaQueries: PropTypes.any,
-    // eslint-disable-next-line react/forbid-prop-types
-    breakpointsMax: PropTypes.any,
+    breakpoints: PropTypes.shape({
+        xs: PropTypes.string,
```

Comment:
Would it be possible to have a single shape and re-use it where it's needed?

Reply:
Sure.